Intersect (Core)/Enums/MapAttribute.cs

```diff
@@ -1,29 +1,38 @@
 namespace Intersect.Enums
 {
-    public enum MapAttribute : byte
+    /// <summary>
+    /// This enumeration type represents the attributes and/or entities within in-game tiles.
+    /// </summary>
+    public enum MapAttribute
     {
-        Walkable = 0,
+        Animation,
 
         Blocked,
 
-        Item,
+        Critter,
+
+        Event,
+
+        GrappleStone,
 
-        ZDimension,
+        Item,
```

Comment:
You can't change the order of an enum like this, as by default each enumerated value is assigned a numeric value based off of its order within the enum, i.e `(int)Walkable == 0`, `(int)Blocked == 1, etc`.

If you really wanted to sort these, there needs to be a corresponding migration that re-maps attributes to their new values for any existing map